javascript/javascript1/week3/timeTravel.js

```diff
@@ -0,0 +1,16 @@
+//When will we be there??
+
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+  
+  function getTravelTime(travelInformation){
+    
+    return Math.trunc(travelInformation.destinationDistance * 60/travelInformation.speed);
+  }
+
+  const travelTime = getTravelTime(travelInformation);
+  const timeInHours = Math.floor(travelTime/60);
+  const timeInMinutes = travelTime % 60;
```

Comment:
These calculations should be inside the function, so all it returns is 
```javascript
`Travel time: ${timeInHours} hours and ${timeInMinutes} minutes`
```

Reply:
Yeah. I did not check that before- I changed it now